src/server/config.js

```diff
@@ -51,17 +36,14 @@ function loadBabelConfig(babelConfigPath) {
 export default function (configType, baseConfig, configDir) {
   const config = baseConfig;
 
-  // Search for a .babelrc in the config directory, then the module root
+  // Search for a .babelrc starting in the config directory
   // directory. If found, use that to extend webpack configurations.
-  let babelConfig = loadBabelConfig(path.resolve(configDir, '.babelrc'));
-  let inConfigDir = true;
-
-  if (!babelConfig) {
-    babelConfig = loadBabelConfig('.babelrc');
```

Comment:
Here we are trying to load two babelConfig files? First try that in the storybook config dir and then in the project root.

I think that logic is not done in this new setup.

> We've no idea where the configDir is located. By default it's `.storybook` but usually it's getting changed.